R/pairwise-comparisons.R

```diff
@@ -53,8 +52,8 @@
 #'
 #' eval <- scoringutils::eval_forecasts(scoringutils::range_example_data_long)
 #' pairwise <- pairwise_comparison(eval, summarise_by = c("model"))
-#' @author Nikos Bosse \email{nikosbosse@gmail.com}
-#' @author Johannes Bracher, \email{johannes.bracher@kit.edu}
+#' @author Nikos Bosse \email{nikosbosse@@gmail.com}
+#' @author Johannes Bracher, \email{johannes.bracher@@kit.edu}
```

Comment:
I think on review this is intentional? I had no idea was needed.

Reply:
Yep :), https://roxygen2.r-lib.org/articles/rd-formatting.html#links-1 (can't find the official source right now). As far as I know, it works well with a single `@` in most cases. But this is the 'correct' syntax if we want to follow the rules.